node/src/components/consensus/era_supervisor.rs

```diff
@@ -674,3 +688,43 @@ where
         }
     }
 }
+
+/// Metrics to track rate of finalization
+#[derive(Debug)]
+pub struct EraSupervisorMetrics {
+    /// Gauge to track rate.
+    finalization_time: Gauge,
+    /// Amount of finalized blocks.
+    amount_of_blocks: IntCounter,
```

Comment:
I think we should track it in the linear chain component.

Reply:
I still think this should be tracked in the linear chain component. It is the component that announces linear chain blocks to the world.